CategoriesPanel.c

```diff
@@ -34,13 +36,23 @@ static void CategoriesPanel_delete(Object* object) {
 }
 
 void CategoriesPanel_makeMetersPage(CategoriesPanel* this) {
-   MetersPanel* leftMeters = MetersPanel_new(this->settings, "Left column", this->header->columns[0], this->scr);
-   MetersPanel* rightMeters = MetersPanel_new(this->settings, "Right column", this->header->columns[1], this->scr);
-   leftMeters->rightNeighbor = rightMeters;
-   rightMeters->leftNeighbor = leftMeters;
-   Panel* availableMeters = (Panel*) AvailableMetersPanel_new(this->settings, this->header, (Panel*) leftMeters, (Panel*) rightMeters, this->scr, this->pl);
-   ScreenManager_add(this->scr, (Panel*) leftMeters, 20);
-   ScreenManager_add(this->scr, (Panel*) rightMeters, 20);
+   size_t columns = HeaderLayout_getColumns(this->scr->header->headerLayout);
+   MetersPanel** meterPanels = xMallocArray(columns, sizeof(MetersPanel));
+
+   for (size_t i = 0; i < columns; i++) {
+      char titleBuffer[32];
+      xSnprintf(titleBuffer, sizeof(titleBuffer), "%zu. Column", i + 1);
```

Comment:
This probably should avoid the problem of display of ordinal numbers, as the correct counting would probably be "1st column", "2nd column", "3rd column" and so on.
```suggestion
      xSnprintf(titleBuffer, sizeof(titleBuffer), "Column %zu", i + 1);
```